projects/experimental/pipes/amount/amount.options.ts

```diff
@@ -0,0 +1,26 @@
+import {Provider} from '@angular/core';
+import {TuiCurrencyVariants, TuiMoneySign} from '@taiga-ui/addon-commerce';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk';
+import {TuiDecimal, TuiHorizontalDirection} from '@taiga-ui/core';
+
+export interface TuiAmountOptions {
+    readonly currency: TuiCurrencyVariants;
+    readonly currencyAlign: TuiHorizontalDirection;
+    readonly decimal: TuiDecimal;
+    readonly precision: number;
+    readonly sign: TuiMoneySign;
+}
+
+export const TUI_AMOUNT_DEFAULT_OPTIONS: TuiAmountOptions = {
+    decimal: `not-zero`,
+    currency: null,
+    currencyAlign: `right`,
+    sign: `negative-only`,
+    precision: 2,
+};
```

Comment:
Should we move `decimal`, `sign` and `precision` to that new directive I proposed?